framework/tests/viewpointtest/VarargType.java

```diff
@@ -0,0 +1,27 @@
+import viewpointtest.quals.*;
+
+public class VarargType {
+
+    VarargType(String str, Object... args) {}
+
+    void foo() {
+        VarargType a = new VarargType("testStr", new Object());
+    }
+
+    // inner class
+    class Inner {
+        Inner(String str, Object... args) {}
```

Comment:
How about only the varargs parameter and then test with:
- new Inner()
- new Inner(new Object())
- Varargtype.this.new Inner()
- Varargtype.this.new Inner(new Object())

To test more of the tricky combinations with inner classes.

Reply:
Added.